frontend/src/core/l10n/actions.js

```diff
@@ -55,7 +58,13 @@ export function get(locales: Array<string>): Function {
             locales.push('en-US');
```

Comment:
Remove this if here?

Reply:
Is `negotiateLanguage` taking care of it? If not, why do you now say we shouldn't enforce `en-US` to always be the last fallback language?

Reply:
It does, yes! `{ defaultLocale: 'en-US' }` will make sure that `en-US` is on the list of negotiated languages, if needed.